indicators/NO_BFLY_001/R/NO_BFLY_001.qmd

```diff
@@ -0,0 +1,867 @@
+        .default = NA_character_
+      ))
+tmap::tmap_mode("plot")
+tm_shape(counties3) + 
+  tm_polygons(col = "Region") 
+
+# Merge counties from same region
+
+counties4 <- counties3 |>
+  group_by(Region) |>
+  summarize(geom = st_union(geom))
+
+# Take out the marine areas
+# import outline of Norway
+nor <- sf::st_read(here::here("indicators/NO_BFLY_001/data/outlineOfNorway_EPSG25833.shp"),
+  quiet = TRUE)
+# remove marine areas
+counties <- sf::st_intersection(counties4, nor)
+
+tm_shape(counties) + 
+  tm_polygons(col = "Region") 
+
+# Save
+saveRDS(counties, here::here("indicators/NO_BFLY_001/data/regions.rds"))
+```
+
+```{r fig-regions}
+#| fig-cap: 'Regions in Norway'
+
+readRDS(here::here("indicators/NO_BFLY_001/data/regions.rds")) -> counties
+tm_shape(counties) + 
+  tm_polygons(col = "Region") +
+  tm_layout(legend.position = c("right", "bottom"))
+```
+
+## 8. Spatial units
+
+```{=html}
+<!--# 
+
+Describe the spatial units that you rely on in your analyses. Highlight the spatial units (the resolution) that the indicator values should be interpretted at. Potential spatial delineation data should eb introduced under 7.1. Datasets. We recomend using the SEEA EA terminology opf Basic Spatial Units (BSU), Ecosystem Asses (EA) and Ecosystem Accounting Area (EAA). 
+
+-->
+```
+
+Ecosystem assets: Mid-, central-, east-, and southern Norway. Note that southern Norway only includes the old counties of Vestfold and Østfold, and the south region contain Rogaland and Vest Agder, but not Øst Agder. The mid or central region includes Trøndelag and Møre og Romsdal.
+
+Ecosystem accounting area: Norway (although N-Norway and several sub-regions are missing).
+
+Data needs to be aggregated to EA level before interpretation. In addition, only locations with complete sampling (three replicated per season) should be included to avoids seasonal biases.
+
+## 9. Analyses
+
+```{=html}
+<!--# 
+
+Use this header for documenting the analyses. Put code in seperate code chunks, and annotate the code in between using normal text (i.e. between the chunks, and try to avoid too many hashed out comments inside the code chunks). Add subheaders as needed. 
+
+Code folding is activated, meaning the code will be hidden by default in the html (one can click to expand it).
+
+Caching is also activated (from the top YAML), meaning that rendering to html will be quicker the second time you do it. This will create a folder inside you project folder (called INDICATORID_cache). Sometimes caching created problems because some operations are not rerun when they should be rerun. Try deleting the cash folder and try again.
+
+-->
+```
+
+These calculations with bootstrap sampling are implemented in the R-package [bombLepiSurv](https://github.com/NINAnor/bombLepiSurv).
+
+Install package like this.
+
+```{r}
+#| eval: false
+#| code-fold: show
+remotes::install_github("NINAnor/bombLepiSurv")
+library(bombLepiSurv)
+```
+
+Connect to the database
+
```

Comment:
I can only run and check the code until here. I do not have access to the database. Is the data small enough to get a copy and try to run the code?